docs/rules/no-template-target-blank.md

```diff
@@ -0,0 +1,99 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-template-target-blank
+description: disallow target="_blank" attribute without rel="noopener noreferrer"
+---
+# vue/no-template-target-blank
+> disallow target="_blank" attribute without rel="noopener noreferrer"
+
+## :book: Rule Details
+
+This rule disallows target="_blank" attribute without rel="noopener noreferrer".
```

Comment:
Like the description of eslint-plugin-react, could you please add an explanation of how this rule is as useful?

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

Reply:
Thank you for reviewing. I fixed:smile: